test/test.org.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var Github = require('../src/github.js');
+var testUser = require('./user.json');
+var github, org;
+
+describe('Github.Organization', function() {
+   before(function() {
+      github = new Github({
+         username: testUser.USERNAME,
+         password: testUser.PASSWORD,
+         auth: 'basic'
+      });
+      org = github.getOrg();
+   });
+
+   it('should check organization membership', function(done) {
+      org.checkMembership('teenscode', 'daconex', function(err) {
```

Comment:
Please, use some organization we have control on, such as [GitHub Tools](https://github.com/github-tools) and members of our staff.